src/corehost/common/pal.h

```diff
@@ -54,12 +54,15 @@
 #if defined(_WIN32)
 #define LIB_PREFIX
 #define MAKE_LIBNAME(NAME) (_X(NAME) _X(".dll"))
+#define BASE_HOST_RID _X("win")
```

Comment:
@eerhardt Thinking a bit more, what happens if OSX 10.13 (or win11) is incompatible with its predecessors and breaks previous assets? What would be the highest supported RID the future host would support?